packages/vscode-extension/src/handlers.ts

```diff
@@ -716,8 +716,6 @@ export async function connectExistingApiHandler(args?: any[]): Promise<Result<nu
 
   const res = await runUserTask(func, TelemetryEvent.ConnectExistingApi, true);
   if (!res.isOk()) {
-    showError(res.error);
-    ExtTelemetry.sendTelemetryErrorEvent(TelemetryEvent.ConnectExistingApi, res.error);
```

Comment:
Can we keep the telemetry here? since it makes us easier to query our telemetries.

Reply:
this code for vscode, vscode already done this job.